src/server/components/charts-engine/controllers/run.ts

```diff
@@ -87,6 +88,7 @@ export const runController = (
                 headers: {
                     ...res.locals.subrequestHeaders,
                     ...ctx.getMetadata(),
+                    ...Utils.pickZitadelHeaders(req),
```

Comment:
`runController` has several calls to `us` and `bi` (data requests), we need to pass new headers to all of them.

Reply:
Investigated them and covered missed headers